testdriver/datasets.py

```diff
@@ -168,13 +168,31 @@ class CppVersion(Enum):
 class ICU4XVersion(Enum):
   ICU4XV1 = "1.0"
 
+# Versions for known executors
+# TODO: combine the version info
+IcuVersionToExecutorMap = {
+    'node': {
+        '73': ["20.1.0"],
+        '72': ['18.14.2'],
+        '71': ['18.7.0', '16.19.1'],
+        '70': ['14.21.3']
+    },
+    'icu4x': {
+        '71': ['1.0'],
+    },
+    'dart': {},
+    'icu4c': {},
+    'icu4j': {},
 
+}
 # What versions of NodeJS use specific ICU versions
 # https://nodejs.org/en/download/releases/
 NodeICUVersionMap = {
-    "18": "72.1",
-    "16": "71.1",
-    "14": "70.1",
+    "20.1.0": "73.1",
```

Comment:
suggestion: express derivative data as a transformation via code

With the correct structuring of `IcuVersionToExecutorMap`, now `NodeICUVersionMap` should be derivative. According to [this SO answer](https://stackoverflow.com/questions/483666/reverse-invert-a-dictionary-mapping), we should be able to rewrite this as 

```python
NodeICUVersionMap = {v: k for k, v in IcuVersionToExecutorMap['node'].items()}
```